docs/integrations/microsoft-azure/azure-functions.md

```diff
@@ -47,16 +135,176 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
+
+#### Activity Logs
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). If you are already collecting activity logs for a subscription, do not perform this step.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
+:::
+
+##### Enabling Microsoft Defender for Cloud
+
+:::note
+For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/azure-sql/database/azure-defender-for-sql?view=azuresql#enable-microsoft-defender-for-sql). If you have an existing settings, click **Edit Settings**.
+:::
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Edit-Settings.png')} alt="Edit Settings" style={{border: '1px solid gray'}} width="800" />
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Microsoft-Cloud-Defender-Plans-AppService.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
+
+##### Enabling Health Check Metric
+
+:::note
+For getting health check metric, make sure you enable health check by going to Monitoring tab.
+:::
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/Enable-Health-Check-Metric.png')} alt="Cloud Defender Plans" style={{border: '1px solid gray'}} width="800" />
+
+## Installing the Azure Functions app
+
+This section provides instructions on how to install the Azure Functions app, and shows examples of each of the preconfigured dashboards you can use to analyze your data.
+
+import AppInstall2 from '../../reuse/apps/app-install-v2.md';
+
+<AppInstall2/>
+
+## Viewing the Azure Functions dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Overview
+
+The **Azure Functions - Overview** dashboard provides information of any service health incidents or resource health events associated with Azure Functions in your azure account.
+
+Use this dashboard to:
+* View recent resource and service health incidents.
+* View distribution of service and resource health by incident type.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-SQL/Azure-SQL-Health.png')} alt="Azure SQL health dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Cost
+
+The **Azure Functions - Cost** dashboard provides information on the expenses associated with your Azure Functions. This includes details on the cost of resources, usage trends, and cost management insights.
+
+Use this dashboard to:
+* Monitor and analyze your spending on Azure Functions.
+* Review cost trends and identify areas where you can optimize spending.
+* Access detailed billing information and cost breakdowns.
+
```

Comment:
Don't we have dashboard images for rest of the dashboards?

Reply:
yes we do have, @ankurch627  will update the urls